### PR TITLE
Add unittest skeleton for GeoAPI

### PR DESCRIPTION
**Pull request by jblomer**
@DrDaveD here's my take on the Python unit tests.  They can be run with `ptyhon -m unittest test_cvmfs_geo`.  New tests can be dropped as additional methods in `GeoTest`.

I'll leave the pull request open over the weekend; others with more Pyhton experience might want to comment.

### CODE REVIEW

**Comment by DrDaveD** [comment]
@jblomer I'm very glad you did this because I wouldn't have known what to do.

Now when I run 'python -m unittest test_cvmfs_geo' on el7, it says 'Ran 1 test in 0.000s', but if I run it on el6, it says 'Ran 0 tests in 0.000s'.  Do you have any idea why?

**Comment by jblomer** [comment]
It seems to work better with `python test_cvmfs_geo.py`.

File: ci/run_travis.sh [hunk 1]
```diff
@@ -82,6 +82,7 @@ echo -n "RUNNING UNIT TESTS "
 if running_on_linux; then
   start_processing
   test/unittests/cvmfs_unittests --gtest_shuffle --gtest_filter="-*Slow:T_Dns.CaresResolverLocalhost:T_Dns.NormalResolverCombined:T_Dns.CaresResolverMany" > unittests.log 2>&1  || { UNITTESTS_FAILED=true; report_error "$(pwd)/unittests.log"; }
+  (cd cvmfs/webapi && python test_cvmfs_geo.py) >> unittests.log 2>&1 || { UNITTESTS_FAILED=true; report_error "$(pwd)/unittests.log"; }
```
Comment (DrDaveD) [inline ci/run_travis.sh]: This needs to be cd ../cvmfs/webapi.  I merged this pr into pr #1983, made that change to this line and also added a -v after test_cvmfs_geo.py, and it passed a [travis test](https://travis-ci.org/DrDaveD/cvmfs/jobs/276033753).  My DrDaveD/cvmfs repo is having better luck at getting travis jobs completed right now than cvmfs/cvmfs; there appears to be a lot queued up here.

Comment (DrDaveD) [reply ci/run_travis.sh]: The travis holdup is the osx builds.  Mine is also blocked on that.  There are four later cvmfs/cvmfs builds pending on both platforms right now, waiting for one build to finish its osx.

**Pull request merged**